lib/warder/coffee_lint_runner.rb

```diff
@@ -0,0 +1,60 @@
+require 'coffeelint'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.